src/News Articles/NewsArticles.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { FunctionComponent, useState, useEffect } from "react";
+import styles from './NewsArticles.module.scss';
+
+export interface Article {
+    imageUrl?: string | undefined;
+    title: string;
+    articleUrl: string;
+    summary?: string | undefined;
+    publicationDate?: string | undefined;
+}
+
+interface NewsArticleProps {
+    article: Article;
+}
+
+const fetchNewsArticles = async (): Promise<Article[]> => {
+
+    return Promise.resolve([
+        {
+            imageUrl: "http://mars.jpl.nasa.gov/msl-raw-images/proj/msl/redops/ods/surface/sol/01000/opgs/edr/fcam/FLB_486265257EDR_F0481570FHAZ00323M_.JPG",
+            title: "title",
+            articleUrl: "articleUrl",
+            summary: "summary",
+            publicationDate: "date"
+        }
+    ]);
+}
+
+const NewsArticle: FunctionComponent<NewsArticleProps> = ({article}) => {
+    return (
+        <div className={styles.indivArticle}>
+            
+            <img alt="" src={article.imageUrl} />
+            <div>
+                <h4 className={styles.articleTitle}>{article.title}</h4>
+                <p>{article.summary}</p>
+                <p>{article.publicationDate}</p>
+                <a href={article.articleUrl}>Read now</a>
+            </div>
```

Comment:
imageUrl, articleUrl and publicationDate might not always be there (they are optional)
Does this still work if they are missing?